conda_smithy/schema.py

```
@@ -12,7 +12,7 @@
 from conda.base.constants import KNOWN_SUBDIRS
 
 try:
-    from enum import StrEnum
+    from enum import StrEnum # type: ignore
 except ImportError:
     from backports.strenum import StrEnum
```

I think it's better to ignore the error on the second import, at least that’s what I gather from [here](https://stackoverflow.com/a/59060931).

---

hmm moving the ignore to the second import StrEnum brings back the `error: Module "enum" has no attribute "StrEnum"` :(